io.github.essentialist_app.essentialist.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.github.essentialist_app.essentialist
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: essentialist
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
```

Comment:
why does it need all of home?

Reply:
The app needs to read and write files in the directory containing the flashcards, this directory can be anywhere in the home directory. If this is recommended, I can set it to `--filesystem=xdg-documents` and constraint the users to put their decks over there. Is it better?

Reply:
Thanks. I have removed the home permission.